### PR TITLE
Update Ice/minimal demos 

### LINKED ISSUE #197: Upgrade Ice/minimal examples
We should align them with C# updated in https://github.com/zeroc-ice/ice-demos/pull/195

### PR DESCRIPTION
**Pull request by pepone**
Fix #197

### CODE REVIEW

File: cpp11/msbuild/ice.proj [hunk 1]
```diff
@@ -7,10 +7,29 @@
 
     <!-- Custom task to download files -->
     <!-- Nuget executable -->
-    <PropertyGroup>
-      <NugetExe>$(MSBuildThisFileDirectory)NuGet-4.9.2.exe</NugetExe>
-      <NugetURL>https://dist.nuget.org/win-x86-commandline/v4.9.2/nuget.exe</NugetURL>
-    </PropertyGroup>
+    <Choose>
```
Comment (externl) [inline cpp11/msbuild/ice.proj:10]: Do other proj files need these updates too?

**Review by externl — APPROVED** [review]
Looks good to me!

File: cpp11/msbuild/ice.proj (hunk 1 above)
Comment (pepone) [reply cpp11/msbuild/ice.proj:10]: Need to check C#

**Review by bernardnormier — APPROVED** [review]
Looks good.

**Pull request merged**